src/models/show.model.js

```diff
@@ -55,6 +55,10 @@ export default function (sequelize, DataTypes) {
             type: DataTypes.STRING(5),
             allowNull: false,
             defaultValue: 'no'
+        },
+        digasId: { // For use with LDAP
```

Comment:
Nothing to do with LDAP, "Digas" is the program used by radio journalists to manage the archive of podcasts and ondemand streams.